### ISSUE TITLE
hdf5.h not found when compiling hlhdf on Mac OS

### ISSUE THREAD

**Issue by adokter** [adokter/vol2birdinstall#2]
The current installer doesn't install or locate the hdf5 library:
```
-------Compiling directory hlhdf------------
gcc   -DBYTE_ORDER_LITTLE_ENDIAN    -DDEBUG_HLHDF   -c -o hlhdf.o hlhdf.c
In file included from hlhdf.c:26:
hlhdf.h:28:10: fatal error: hdf5.h: No such file or directory
   28 | #include <hdf5.h>
      |          ^~~~~~~~
compilation terminated.
make[1]: *** [hlhdf.o] Error 1
```
When installing hlhdf manually, I always need to give it this install flag (having installed hdf5 with macports, not homebrew):
```
 --with-hdf5=/opt/local/include,/opt/local/lib
```

**Comment by adokter** [comment]
closed by commit efea0ce

**Comment by MichevB** [comment]
When building on a fresh Linux Mint (21.3) installation the installer returns a similar error message: 

-------Compiling directory hlhdf------------
make[1]: Entering directory '/home/ecotan/Documents/vol2birdinstall/.build/hlhdf/hlhdf'
gcc  -fPIC -DBYTE_ORDER_LITTLE_ENDIAN       -c -o hlhdf.o hlhdf.c
In file included from hlhdf.c:26:
hlhdf.h:28:10: fatal error: hdf5.h: No such file or directory
   28 | #include <hdf5.h>
      |          ^~~~~~~~
compilation terminated.
make[1]: *** [<builtin>: hlhdf.o] Error 1
make[1]: Leaving directory '/home/ecotan/Documents/vol2birdinstall/.build/hlhdf/hlhdf'
make: *** [Makefile:46: all] Error 255
(HLHDF) Failed to compile software
